mappings/net/minecraft/world/World.mapping

```diff
@@ -15,7 +15,7 @@ CLASS net/minecraft/class_101 net/minecraft/world/World
 	FIELD field_299 diskSize J
 	FIELD field_300 illuminatedAreas Ljava/util/List;
 	FIELD field_301 tickableEntities Ljava/util/List;
-	FIELD field_302 treeSet Ljava/util/TreeSet;
+	FIELD field_302 worldTreeSet Ljava/util/TreeSet;
```

Comment:
What does this tree set manage though? Does it manage ex: removed entities? worldTreeSet isn't very descriptive of what it does functionally.